index.md

```diff
@@ -568,9 +568,9 @@ contact the <a href="mailto:featureide@uni-ulm.de">FeatureIDE development team</
 <li>Patrik Haese (University of Magdeburg, Germany)</li>
 <li>Philipp Kuhn (TU Braunschweig, Germany)</li>
 <li>Philipp Vulpius (University of Ulm, Germany)</li>
-<li><a href="https://wwwiti.cs.uni-magdeburg.de/~rschroet/">Reimar Schr&ouml;ter</a> (University of Magdeburg, Germany)</li>
+<li>Reimar Schr&ouml;ter (University of Magdeburg, Germany)</li>
```

Comment:
This is unfortunate that you put websites offline in Magdeburg. Please at least link to former members if the page does not exist anymore.